misc/snapshotter/entrypoint.sh

```diff
@@ -17,10 +17,7 @@ if [ "$#" -eq 0 ]; then
 		--root ${NYDUS_LIB} \
 		--address ${NYDUS_RUN}/containerd-nydus-grpc.sock \
 		--log-level ${LEVEL} \
-		--enable-metrics=${ENABLE_METRICS} \
```

Comment:
Shall we remove the related variables as well?
```shell
ENABLE_NYDUS_OVERLAY="${ENABLE_NYDUS_OVERLAY:-true}"
ENABLE_METRICS="${ENABLE_METRICS:-false}"
```

Reply:
Good point, already removed